apis/meta/annotations.go

```diff
@@ -22,4 +22,52 @@ const (
 	// interpreted as a timestamp, and any change in value shall trigger a
 	// reconciliation.
```

Comment:
```suggestion
	// reconciliation.
	// DEPRECATED: has been replaced by ReconcileAtAnnotation.
```

This makes GoLand (and other "intelligent" IDEs) show it with strikethrough markup.